src/scss/survey.scss

```diff
@@ -0,0 +1,96 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import 'includes/mixins';
+@import 'includes/variables';
+
+.survey-banner {
+    @include gradient-horizontal($start-color: #0001fe, $end-color: #2d1144);
+    bottom: 0;
```

Comment:
I'm gonna make the same suggestion I did on IRL, to pad the bottom and do a negative bottom position so it continues past the viewport when it pops up.